00_Base/src/interfaces/messages/index.ts

```diff
@@ -32,6 +32,15 @@ export enum EventGroup {
   Reporting = 'reporting',
   SmartCharging = 'smartcharging',
   Transactions = 'transactions',
+  Cdrs = 'cdrs',
```

Comment:
We might want to prefix these with "Ocpi".

E.g. OcpiCdrs = 'OcpiCdrs'

Reply:
perhaps that can be done in a followup PR?